### ISSUE TITLE
logopen does not create file in 5.x

### ISSUE THREAD

**Issue by simonztan** [TeraTermProject/teraterm#204]
Hi,

I've been using this below command in my script for awhile to create log file which works in Teraterm 4.x but once upgraded to 5.x, it does not seems working

logopen logfile 0 1 0 0 1

Is there any change in 5.x on the way the file handle?

**Comment by simonztan** [comment]
below is the sample of the script

;## Global setting ##

username = 'admin'       ; username used to login into Fortigate (super_admin) profile is require
hostname = '10.56.2.204'  ; IP address of the device
sshport = '22'           ; change if using custom ssh port
dpath = 'C:\IPSEDebug'   ; log file store location
dprocessfile='IPSE'

;######################## DO NOT MAKE ANY CHANGES BELOW EXCEPT DEBUG COMMAND Section ###############################

msg = 'Enter password for user '
strconcat msg username
passwordbox msg 'Enter password'

msg = hostname
strconcat msg ':'
strconcat msg sshport
strconcat msg ' /ssh /auth=password /user='
strconcat msg username
strconcat msg ' /passwd='
strconcat msg inputstr

connect msg

foldercreate dpath
changedir dpath
strconcat dprocessfile '-debug-%Y%m%d.txt'
getdate logfile dprocessfile
logopen logfile 0 1 0 0 1

**Comment by zmatsuo** [comment]
Thank you for your report.
I confirmed.

Try following workaround.

```
dpath = 'C:\IPSEDebug' ; log file store location
foldercreate dpath
strconcat dprocessfile '-debug-%Y%m%d.txt'
getdate logfile dprocessfile
strconcat logfull dpath
strconcat logfull '\'
strconcat logfull logfile
logopen logfull 0 1 0 0 1
```

- On Tera Term 4, `logopen` create file in download folder
  if default log save folder is empty.
- On Tera Term 5, `logopen` create file in [log folder](https://teratermproject.github.io/manual/5/en/setup/folder.html#LogDir)
  if default log save folder is empty.
- `changedir` macro command changes download folder

This seems not good. I will fix .

**Comment by zmatsuo** [comment]
Fixed.

Please try this binary.
https://ci.appveyor.com/project/teraterm/github-main/builds/49703205/artifacts

**Comment by simonztan** [comment]
I've tested the binary and it works well, thanks for your quick fix. I believe it should be fix in 5.3?

**Comment by zmatsuo** [comment]
Thank you.😀
This fix will be included in 5.3 
This issue is closed.
